js/script.js

```diff
@@ -69,24 +69,45 @@ const parkingLotsStyle = {
   fillOpacity: 0.6,
 };
 
-const determineAndSetUpLocationTag = (url) => {
-  let result = "";
-
-  if (url.indexOf("#parking-reform-map") > -1) {
-    const [, splitAddressFragment] = url.split("#");
-    // If the tag is "parking-reform-map=....."
-    if (splitAddressFragment.indexOf("parking-reform-map") > -1) {
-      $("#toggle-projects-by-tag").css("display", "none");
```

Comment:
This line was moved out of the function and into `setUpSite()`. It's a "side-effect" (depends on external state) and would cause the unit test to fail.